src/main/webapp/WEB-INF/static/js/trialmanager/experimentalDesign.js

```diff
@@ -276,7 +276,7 @@
 								});
 							} else {
 								showAlertMessage('', 'All instances cannot be regenerated due to internal validations (presence of ' +
-									'samples or inventory transactions or subobservations or advance/cross list).');
+									'samples, inventory transactions, subobservations, means or advance/cross list).');
```

Comment:
expand as "means **data**"

Reply:
Noted. Also applied in Messages.properties: environment.cannot.be.deleted